python/py-isort/files/isort27

```diff
@@ -0,0 +1 @@
+bin/isort-2.7
```

Comment:
please change the actual select-files (here and for the other ones as well) to point to the "bin" directory in the Python framework:

```
${frameworks_dir}/Python.framework/Versions/2.7/bin/isort-2.7
${frameworks_dir}/Python.framework/Versions/2.7/bin/isort-identify-imports-2.7`
```

to be hones I am not completely sure that what you had wouldn't work, but this is how it happens in most other ports (so there is likely a reason for it and otherwise there is a least value in consistency).

Reply:
It looks to be fairly split. `black`, `mypy`, `pytest`, `tox`, and `flake8`, among others, all link directly to `bin`, while `sphinx`, `pylint`, `yapf` and others link to the Python.framework dir.

I don't know what the "right" approach is, and I don't have a preference either way.

Reply:
ah, I see - I just looked at a few ports and those all seemed to use the Python.framework dir. But in retrospect I don't think it matters because the files are identical (e.g., `bin/isort-3.9` and `${frameworks_dir}/Python.framework/Versions/3.9/bin/isort`).